concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -1300,13 +1334,23 @@ jobs:
           - get: cf-manifest
             passed: ['cf-deploy']
           - get: bosh-CA
+          - get: cf-secrets
+            passed: ['cf-deploy']
+
+      - task: create-temp-user
+        file: paas-cf/concourse/tasks/create_admin.yml
```

Comment:
Do the custom acceptance tests need a user with admin privileges? Can we create a less privileged user?